### DISCUSSION TITLE
Extra password or fingerprint for better security

### DISCUSSION THREAD

**Discussion by agostaxyz** [tutao/tutanota#3240] [Ideas]
It could be some sort of native Android/IOS implementation, or maybe an additional PIN to unlock the app.  
  
I live in Brazil, here phone theft and pickpocketing is very common, I'm linking an article in English about this issue, criminals can use the email the user may have auto signed in to access "Forgot my password" on other apps, so it could basically gain access to ALL the user apps, including personal photos, social media and banking, I also know cases where the criminals make you unlock your phone while robbing you, so even if your phone has a default 100 digit password or fingerprint, criminals will bypass it and gain access to your apps.  
    
"(...) criminals appear to have impersonated a victim after breaking into their email account and convinced their bank to transfer thousands of dollars to outside accounts."  
https://gizmodo.com/pickpockets-in-brazil-are-reportedly-going-after-smartp-1847127103

I'm using Brazil as an example but this could be a nightmare to anyone anywhere

**Comment by vitoreiji** [comment]
That's already on the roadmap #545 :-)
